codewars4.py

```diff
@@ -0,0 +1,25 @@
+# https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/python
+
+# def get_count(sentence):
+#     pass
+
+vowels_count = 0
+
+sentence = input("Please write something: ")
+
+for char in sentence:
+    if char.casefold() == "a" or char.casefold() == "e" or char.casefold() == "i" or char.casefold() == "o" or char.casefold() == "u":
```

Comment:
First of all: congratulations on passing this exercise! Great job!

However, there is room for improvement in this if statement. My first thought and suggestion would to declare a list before the loop with all the vowels. For example vowels = [a, e, i, o, u]. Later inside the loop you can simply use if char.casefold() in vowels. This checks if a given character is in the list containing our vowels.

Reply:
Thank you so much for your insight!

I agree and I did as you suggested. I was also thinking of simply defining a string with all the vowels, but a list seems more suitable.

Reply:
I have to say that in my opinion both options are really suitable. Typing a string is probably a bit easier than a list, but both should work equally effective.